Allow Element Call to be started without audio / video interface (#924)
vector-im/element-call: README.md

# edit public/config.json
```

#### Allow to join group calls without camera and audio device:

You can allow to join a group call without video and audio by setup this feature in your config.json:

```json
{
  ...

  "features": {
    "feature_group_calls_without_video_and_audio": true
  }
}
```

> 

Reviewer: Could this be moved to the end of the `Host it yourself` section? Also it would be good to have this under a `Features` section or similar

Therefore, to use a self-hosted homeserver, this is recommended to be a new server where any user account created has not joined any normal rooms anywhere in the Matrix federated network. The homeserver used can be setup to disable federation, so as to prevent spam registrations (if you keep registrations open) and to ensure Element Call continues to work in case any user decides to log in to their Element Call account using the standard Element app and joins normal rooms that Element Call cannot handle.

### Features

> #### Allow to join group calls without camera and audio device
> 
> You can allow to join a group call without video and audio by setup this feature in your `config.json`:

Reviewer: ```suggestion
#### Allow joining group calls without a camera and a microphone

You can allow joining a group call without video and audio enabling this feature in your `config.json`:
```
